components/SpotlightShare/styles.css

```diff
@@ -0,0 +1,28 @@
+#urllink {
+  border-radius: 0.25em;
+  padding: 0.5em 1.5em;
+  border-width: 2px;
+  border-style: solid;
+  outline: 0px;
+  border-color: rgb(42, 39, 41);
+  color: rgb(42, 39, 41);
+  line-height: inherit;
+  height: 3em;
+  margin-top: 2em;
+  width: 100%;
+  display: inline-block;
+}
+.ui.icon.button {
+  padding: 0.785714em;
+}
+.ui.button {
+  cursor: pointer;
+  display: inline-block;
+  min-height: 1em;
+  vertical-align: baseline;
+  background-color: rgb(224, 225, 226);
+  color: rgba(0, 0, 0, 0.6);
+  border-radius: 0.285714rem;
+  border-style: none;
+  line-height: 1em;
+}
```

Comment:
Unless it's a limitation of a library we're using, we shall always use NextJS' css in JS. Replace this with that 
https://nextjs.org/docs/basic-features/built-in-css-support#css-in-js

Reply:
> Nice work. I think this can be fully functional even before being able to modify the URL.
> Any update from Ben on the modal?

No updates from him yet